Blish HUD/GameServices/ArcDps/Models/Ev.cs

```diff
@@ -1,36 +1,117 @@
 ﻿namespace Blish_HUD.ArcDps.Models {
 
     /// <summary>
-    ///     Infos and data about the combat event. Have a look at the arcdps plugin documentation for how to use it
+    ///     Infos and data about the combat event. For more information see the arcdps plugin documentation.
     /// </summary>
     public class Ev {
 
+        /// <summary>
+        /// Time when the event was registered.
```

Comment:
We should note here that the time is some windows specific time since system boot, not the actual time in a known format. You can compare and convert it by looking up the windows docs for that timestamp.